index.html

```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
           </ul>
         </div>
         <div class="space"></div>
+        <div>
+          <img
+            class="drop-down"
+            src="Images/drop-down.png"
+            alt="drop-down button"
+          />
+        </div>
+      </section>
+    </main>
+    <section class="rest">
+      <h1 class="works">My Recent Works</h1>
+      <div class="indicator">
+        <img id="line" src="Images/Indicator.png" alt="fancyLine"/>
+      </div>
+      <div class="more-space"></div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
```

Comment:
[REQUIRED] You are expected to use grid to arrange the cards in the works section while you use flexbox to arrange the contents of each card but I don't see you using grid